velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1399,4 +1400,16 @@ struct TimeZoneMinuteFunction : public TimestampWithTimezoneSupport<T> {
   }
 };
 
+template <typename T>
+struct ToISO8601FromDateFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<Varchar>& result,
+      const arg_type<Date>& date) {
+    result = DATE()->toString(date);
```

Comment:
We should have a static function inside the DateType class and avoid allocating an object for every call.

Reply:
Oh yes! DateType is a singleton! Thanks.